test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java

```diff
@@ -135,7 +137,7 @@ public void testExpectedClasses() {
     expectSet.put("init", Initialize.class);
     expectSet.put("login-info", LoginProperties.class);
     expectSet.put("manager", ManagerExecutable.class);
-    expectSet.put("master", org.apache.accumulo.manager.MasterExecutable.class);
+    expectSet.put("master", MasterExecutable.class);
```

Comment:
This previously used the fully qualified class name because you can't suppress the deprecation warning in the imports. It should be changed back.

```suggestion
    expectSet.put("master", MasterExecutable.class);
```